### DISCUSSION TITLE
I'm back to the pulsating green dot of death!

### DISCUSSION THREAD

**Discussion by steveharman** [plausible/analytics#825] [Self-Hosted Support]
After eventually getting everything working successfully on server X, I sadly had to re-purpose that machine so have created a fresh install of Plausible on a new VPS.  I am now at the stage where Plausible is installed & accessible on the new server, https is active (HAProxy reverse proxy) and I've updated the Wordpress plugin with details of where my new instance is at. 

I can login to my instance but all I get is the "_Waiting for first pageview...._" animation.  I've tried looking in the browser's dev tools > networking and there don't seem to be any errors shown, but I do get an ever increasing number of "status" entries.  At the time of writing it's at 208 and growing.

Is there anything I can do to try & debug this please?

Thanks

<img width="1443" alt="plausible" src="https://user-images.githubusercontent.com/12557929/110671236-30f80380-81c6-11eb-9aa1-f39f149ed2fd.png">

**Comment by ukutaht** [comment]
On this screen the frontend just polls the backend to see if any pageviews for this domain have come through.

Better check the network tab on the page you've installed the script on. Do you see a POST to `yourplausible.com/api/event` ?

**Reply by steveharman** [reply]
Thanks for the reply.  Everything still looks fine (to me) at the frontend & backend, yet I still get "Waiting for first pageview" in the backend web UI.  And I've done some page views.

My backend is on a different domain to the Wordpress install where the Plausible plugin is installed - could that be relevant?  Both the backend & frontend domains are secured by https

<img width="1783" alt="PLAUSIBLE" src="https://user-images.githubusercontent.com/12557929/110773479-b0341880-8254-11eb-8b1f-23b28d9067d1.png">

**Reply by ukutaht** [reply]
Your plausible is installed on `stats.mywire.org` but the script is configured to send the stats to `stats.livewire.org`. Can you double check your `BASE_URL` in the config is correct?

**Reply by steveharman** [reply]
Argh, good spot, thanks.  Sadly though, ensuring the base url is now correct in the docker config (and everywhere else) hasn't helped.    

I've tried restarting the containers, removing, re-adding & re-activating the Wordpress plugin, editing & re-saving the url in the plugin's "Self hosted" tab - everything I can think of.

<img width="1863" alt="plausible" src="https://user-images.githubusercontent.com/12557929/110794085-4411de80-826d-11eb-9bcf-c39c468af2ed.png">

**Comment by bayerlse** [comment]
Hey @steveharman , did you manage to solve the issue? Getting the same error with same context:

- events sent successfully to correct domain
- no problems in db or event-db logs

but just a green pulsating dot....

br, sebi

**Reply by jkrauss82** [reply]
I have the exact same problem.

- no errors in js console
- event POST is successful with a 202 response code from our self-hosted plausible instance
- green dot of death keeps pulsating

~~I wonder if it might have to do with different top level domains? But then I would expect to get a CORS error of some kind for instance.~~

~~Anyone got any idea what's going on?~~

I have found the issue: we were filtering traffic from our own IPs on the host of our plausible instance. When using another IP I can see the events coming in.

**Comment by steveharman** [comment]
Afraid my "solution" was to use something else.  Sorry.  ;-)
